dashboard/app/bisect_test.go

```diff
@@ -1232,10 +1192,25 @@ func TestBugBisectionStatus(t *testing.T) {
 		},
 	}
 	c.expectOK(c.client2.JobDone(done))
-	content, err = c.GET(url)
-	c.expectEQ(err, nil)
-	c.expectTrue(bytes.Contains(content, []byte("done")))
-
 	msg := c.client2.pollEmailBug()
 	c.expectTrue(strings.Contains(msg.Body, "syzbot suspects this issue was fixed by commit:"))
+	return resp, done, jobID
+}
+
+func getBug(c *Ctx) (*Bug, *db.Key) {
```

Comment:
Nit: these might be good candidates for `Ctx` methods in `util_test.go`. `getSingleBug`/`getSingleJob`?